public/static/template/mam.js

```diff
@@ -0,0 +1,73 @@
+const { asciiToTrytes } = require('@iota/converter');
+const {
+  createChannel,
+  createMessage,
+  mamAttach
+} = require('@iota/mam.js');
```

Comment:
Please also consider updating the `public/static/template/package.json` file to include the new lib and to remove legacy ones